editor/deprecated.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import { forEach } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { deprecated } from '@wordpress/utils';
+
+import {
+	Autocomplete,
+	AlignmentToolbar,
+	BlockAlignmentToolbar,
+	BlockControls,
+	BlockEdit,
+	BlockFormatControls,
+	BlockIcon,
+	ColorPalette,
+	ContrastChecker,
+	ImagePlaceholder,
+	InnerBlocks,
+	InspectorAdvancedControls,
+	InspectorControls,
+	PlainText,
+	RichText,
+	RichTextProvider,
+	MediaUpload,
+	UrlInput,
+	UrlInputButton,
+	withColorContext,
+	getColorClass,
+	getColorName,
+	withColors,
```

Comment:
@jorgefilipecosta what's the difference between `withColors` and `withColorContext`?
It's the last chance to rename them to make it less confusing.